lib/service.rb

```diff
@@ -101,7 +113,7 @@ def pid?
     def error?
       return false if pid?
 
-      exit_code.blank? || exit_code.nonzero?
+      !(exit_code.present? && exit_code.nonzero?)
```

Comment:
```suggestion
      exit_code.present? && exit_code.nonzero?
```
This is probably what you want then. A missing exit code sounds to me like something went wrong tho.

Reply:
The only place it currently can't find an exit code is when your system restarts on Linux and you haven't started the service yet. So it's not an error, it's "stopped"